packages/react-ui/components/TokenInput/TokenInput.tsx

```diff
@@ -412,15 +440,54 @@ export class TokenInput<T = string> extends React.PureComponent<TokenInputProps<
     } else {
       this.dispatch({ type: 'BLUR' });
     }
+
     if (this.props.onBlur) {
       this.props.onBlur(event);
     }
   };
 
+  private handleOutsideBlur = () => {
+    const { inputValue, autocompleteItems } = this.state;
+    const { valueToString } = this.props;
+
+    if (inputValue === '') {
+      // если стерли содержимое токена в режиме редактирования, то удаляем токен
+      if (this.isEditingMode) {
+        this.finishTokenEdit();
+      }
+      return null;
```

Comment:
```suggestion
      return;
```

Этот null же нигде не нужен. В таких ситуациях обычно просто выходят из функции. 
Еще в паре мест ниже такое же есть.

Reply:
опять же остатки прошлой работы во мне. там линтер не пропускал пустые return

Reply:
![image](https://user-images.githubusercontent.com/4607770/107007145-f1e42600-67b3-11eb-9256-12ae8a5b082e.png)